### PR TITLE
Add typeahead-addresspicker@0.1.4 with git autoupdate

### PR DESCRIPTION
**Pull request by steve0hh**
**Library name: bootstrap-progressbar**
**Git repository url: https://github.com/sgruhier/typeahead-addresspicker**
**License(s): The MIT License (MIT) [found here](https://github.com/sgruhier/typeahead-addresspicker/blob/master/LICENSE)**
**Official homepage: http://sgruhier.github.io/typeahead-addresspicker/**

cc @sgruhier

### CODE REVIEW

**Comment by sgruhier** [comment]
What is git autoupdate ?

**Comment by steve0hh** [comment]
@sgruhier 
> cdnjs automatically updates libraries that are known to be hosted on npm or git repo, e.g., Lodash. npm auto-update relies on each release and git auto-update relies on the tags in git repo. This auto-update script runs every hour, but the update result won't be committed until one of our maintainers audit the updates, because many libs will change the naming or directory structure during different versions, and we may need to minify the lib without pre-minified dist files, so it'll be reasonable to delay for at most 30 hours, if you think there is a lib didn't been updated, please wait at least 30 hours for the process to audit it.

**Comment by PeterDaveHello** [comment]
Please also add license field in package.json, thanks.

File: ajax/libs/typeahead-addresspicker/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "google",
+    "map"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sgruhier/typeahead-addresspicker.git",
+    "basePath": "dist",
+    "files": [
+      "typeahead-addresspicker.js",
+      "typeahead-addresspicker-min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/typeahead-addresspicker/package.json]: Should use `"**/*"` instead of the two lines above.

**Comment by steve0hh** [comment]
add license field and change files to `"**/*"`. :smile:

**Comment by PeterDaveHello** [comment]
Thanks!

**Pull request merged**